### PR TITLE
Update substrate/polkadot + needed changes to compile

### PR DESCRIPTION
**Pull request by gilescope**
Slight changes to emulator needed to cope with latest master.

### CODE REVIEW

File: client/network/src/lib.rs [hunk 1]
```diff
@@ -334,9 +333,9 @@ where
 			let relay_chain_is_syncing = relay_chain_interface
 				.is_major_syncing()
 				.await
-				.map_err(|e| {
-					tracing::error!(target: LOG_TARGET, "Unable to determine sync status. {}", e)
-				})
+				.map_err(
+					|e| tracing::error!(target: LOG_TARGET, "Unable to determine sync status. {}", e),
+				)
 				.unwrap_or(false);
```
Comment (gilescope) [inline client/network/src/lib.rs:339]: These are just cargo fmt changes in this file.

**Comment by gilescope** [comment]
already contained by https://github.com/paritytech/cumulus/pull/2612

**Comment by gilescope** [comment]
As https://github.com/paritytech/cumulus/pull/2612 might not land immediately then we'll use this one to unblock people.

**Review by mrcnski — APPROVED** [review]
Thanks! BTW it looks like the rustfmt job is failing.

**Comment by NachoPal** [comment]
bot fmt

**Review by NachoPal — APPROVED** [review]

**Review by ggwpez — APPROVED** [review]

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**